### PR TITLE
HDDS-10473. Simplify some strings in HddsDispatcher

### PR DESCRIPTION
**Pull request by jianghuazhu**
## What changes were proposed in this pull request?
There are some redundant strings in the HddsDispatcher and the purpose of this pr is to simplify them.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-10473

## How was this patch tested?
You need to guarantee that the unit tests pass.

### CODE REVIEW

**Comment by jianghuazhu** [comment]
@adoroszlai , can you help take a look at this PR?
ci: https://github.com/jianghuazhu/ozone/actions/runs/8169369917

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java [hunk 1]
```diff
@@ -812,43 +811,54 @@ private static DNAction getAuditAction(Type cmdType) {
     }
   }
 
+  private static final String AUDIT_PARAM_CONTAINER_ID = "containerID";
```
Comment (Galsza) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java:814]: By convention the private static final constant fields are declared at the start of the class. Please move these declarations to around line 94.

I think the audit related methods and all these constants could be moved to a separate class/method to make HddsDispatcher shorter. What do you think @adoroszlai ?

**Review by Galsza — COMMENTED** [review]
Thank you for working on this, I've left one minor comment

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java (hunk 1 above)
Comment (jianghuazhu) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java]: Thanks @Galsza  for the comments and review.
I've updated it.

**Comment by Galsza** [comment]
LGTM +1

**Comment by jianghuazhu** [comment]
@adoroszlai , can you help review this pr?
Thanks.

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @jianghuazhu for the patch, @Galsza for the review.

**Pull request merged**